build/vega-lite-schema.json

```diff
@@ -3720,26 +4082,32 @@
       "type": "string"
     },
     "FontWeight": {
-      "anyOf": [
-        {
-          "$ref": "#/definitions/FontWeightString"
-        },
-        {
-          "$ref": "#/definitions/FontWeightNumber"
-        }
-      ]
-    },
-    "FontWeightNumber": {
-      "maximum": 900,
-      "minimum": 100,
-      "type": "number"
-    },
-    "FontWeightString": {
       "enum": [
         "normal",
-        "bold"
+        "bold",
+        "100",
```

Comment:
Why "100", "200" and so on?

Reply:
vega-typings use this. It actually makes sense since 211 is not a supported font weight.

Reply:
See https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

Reply:
But yeah, the strings are weird and I'll fix that.

Reply:
https://github.com/vega/vega-parser/blob/master/schema/encode.js#L5

Reply:
NVM then.

Reply:
It will be fixed the next time we update vega typings